mona_sdk/client.py

```diff
@@ -114,15 +115,22 @@
     "SAMPLING_FACTORS_MAX_AGE_SECONDS", 300
 )
 
-SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service."
+SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service"
 UPLOAD_CONFIG_ERROR_MESSAGE = (
-    "Could not upload the new configuration, please check it is valid."
+    "Could not upload the new configuration, please check it is valid"
+)
+RETRIEVE_CONFIG_HISTORY_ERROR_MESSAGE = "Retrieve history is empty"
+GET_AGGREGATED_STATS_OF_SPECIFIC_SEGMENTATION_ERROR = (
+    "Could not get aggregates state of a specific segmentation"
 )
-APP_SERVER_CONNECTION_ERROR_MESSAGE = "Cannot connect to app-server."
+SAMPLING_FACTORS_EMPTY_MAPPING_ERROR = (
```

Comment:
please add _MESSAGE at the end here and in line 123, and switch the order in line 131 so it all ends with "error message"